terraform/main.tf

```diff
@@ -147,38 +153,73 @@ resource "azurerm_virtual_machine_scale_set" "scaleset" {
     type_handler_version    = "2.0"
     settings                = <<SETTINGS
         {
-         "commandToExecute": "cmd",
+          "commandToExecute": "${var.command}",
           "fileUris": [
-            "https://<storagename>.blob.core.windows.net/<file>",
+            "${var.file1}",

```

Comment:
fileUris should be a variable, what if I add 3rd file?

Reply:
I tried adding a single variable as a JSON array string, but it didn't work. The parser seems to only want to apply variables inside a set of quotes.

It's easy enough to add a third variable if needed, and the number of values would most likely be consistent across multiple environments.